index.html

```diff
@@ -217,5 +217,32 @@ <h4>Skills</h4>
     <hr class="skills">
 </section>
 </main>
+
+<!-- Footer Section -->
+
+<footer class="contact-forms">
+    <div class="contact">
+        <h1 class="form-heading">Contact me</h1> 
```

Comment:
- [ ] Please change this heading to `<h2>` tag instead of `<h1>` for better semantics.